.eslintrc

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
This file shouldn't be committed. Maybe you can add it to .gitignore?

Reply:
Done.